e2e_samples/parking_sensors/devops/templates/jobs/deploy-databricks-job.yml

```diff
@@ -9,6 +9,8 @@ jobs:
     vmImage: 'ubuntu-latest'
```

Comment:
have you considered using the databricks-cli docker image instead? it might be easier as you don't need to install the cli on the vm agent:
the image would be `ghcr.io/databricks/cli:latest`

see https://github.com/databricks/cli?tab=readme-ov-file#docker